src/plugins/intel_gpu/src/runtime/ocl/ocl_stream.cpp

```diff
@@ -274,7 +274,7 @@ void ocl_stream::set_arguments(kernel& kernel, const kernel_arguments_desc& args
         GPU_DEBUG_TRACE_DETAIL << "Set arguments for primitive: " << args_desc.layerID << " (" << kern.get() << ")\n";
         set_arguments_impl(kern, args_desc.arguments, args);
     } catch (cl::Error const& err) {
-        throw ocl_error(err);
+        OPENVINO_THROW(OCL_ERR_MSG_FMT(err));
```

Comment:
could you explain why you changed `ocl_error` to `OCL_ERR_MSG_FMT`? These two look quite similar.

Reply:
sorry but could you explain what is OPENVINO_SUPPRESS_DEPRECATED_START?

Reply:
ocl_error() is deprecated and recommend to use OPENVINO_THROW()

Reply:
OK I did some additional study and let me write that as a record:
* "throw" has additional memory overhead. So the recommendation is to use OPENVINO_THROW. (CVS-103685)